tests/pytests/test_config.py

```diff
@@ -17,6 +17,8 @@ def testConfigErrors(env):
     env.expect('ft.config', 'idx').error().contains('wrong number of arguments')
     env.expect('ft.config', 'set', '_NUMERIC_RANGES_PARENTS', 3) \
         .equal('Max depth for range cannot be higher than max depth for balance')
+    env.expect('ft.config', 'set', 'MINSTEMLEN', 1).error()\
+        .contains('Value is outside acceptable bounds')
```

Comment:
Should reveal the value `MIN_MIN_STEM_LENGHT`

Reply:
Fixed.